marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/matching.sjs

```diff
@@ -78,40 +77,8 @@ function defaultArtifact(artifactName) {
     permissions: defaultPermissions,
     targetFormat: "json"
   };
-
-  if (Security.currentUserHasRole("hub-central-match-merge-writer")) {
-    artifact["matchRulesets"] = artifact.matchRulesets || [];
-    artifact["thresholds"] = artifact.thresholds || [];
-  }
-  else {
-    artifact["matchOptions"] = {
-      dataFormat: "json",
-      propertyDefs: {
-        property: []
-      },
-      algorithms: {
-        algorithm: []
-      },
-      collections: {
-        content: []
-      },
-      scoring: {
-        add: [],
-        expand: [],
-        reduce: []
-      },
-      actions: {
-        action: []
-      },
-      thresholds: {
-        threshold: []
-      },
-      tuning: {
-        maxScan: 200
-      }
-    }
-  }
-
```

Comment:
This change is backing out the changes that I just made for DHFPROD-6178 and -6179, so that the match and merge options are created in either the old or new format depending on the roles of the user. Do we not want to support the creation of QuickStart-compatible steps?

Reply:
While working on test failures in 9.0-x, we realized that when a step is created in QS, it doesn't follow this codepath. This codepath is meant for creating steps either in HC or using hubCreateStep (which creates step in HC format);  so we might as well create the steps in the new format